modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,124 @@ If you have installed the {gitops-title} Operator in the default namespace, run
+
+[id="fixed-issues-1-15-0_{context}"]
+== Fixed issues
+
+* Before this update, when the desired state for an existing Ingress changed the host, the new host information was not applied. This update fixes the issue by adding Ingress checks to ensure the Ingress is updated when the host changes. If there is a change in the Ingress host, it will be correctly updated with the new value, ensuring that the desired state is reflected. link:https://issues.redhat.com/browse/GITOPS-5386[GITOPS-5386]
+
+* Before this update, when the desired state for the server service type changes, for example, from `ClusterIP` to `NodePort`, the change was not applied automatically to the cluster without first deleting the resource. This update fixes the issue by adding a server service check to apply changes to the server service. If the desired state for the server service type changes, it is applied automatically to the cluster without deleting the resource. link:https://issues.redhat.com/browse/GITOPS-5385[GITOPS-5385]
+
+* Before this update, users were unable to set a custom image for the `ApplicationSet` Controller after specifying it in the Argo CD CR. This update fixes the issue by allowing users to set a custom image for the `ApplicationSet` Controller. link:https://issues.redhat.com/browse/GITOPS-5395[GITOPS-5395]
+
+* Before this update, Argo CD was unable to retrieve the correct TLS certificate for Helm OCI registries if the URL contained a path or port number. This update fixes the issue by using a fix in upstream Argo CD that ensures the URL is correctly parsed, allowing Argo CD to retrieve and return a valid TLS certificate. link:https://issues.redhat.com/browse/GITOPS-5081[GITOPS-5081]
+
+[id="deprecated-features-1-15-0_{context}"]
+== Deprecated and removed features
 
 [id="removal-of-application-manager-cli-kam_{context}"]
 === Removal of {gitops-title} Application Manager CLI, kam
 
-* With this release, support for the {gitops-title} Application Manager command-line interface (CLI), `kam`, has been removed because this feature is no longer supported. As a result, all instances of `kam` have been removed from the {gitops-title} documentation. As an alternative to `kam`, you can use the Argo CD CLI that is available from {gitops-title} Operator v1.12.
\ No newline at end of file
+* With this release, support for the {gitops-title} Application Manager command-line interface (CLI), `kam`, has been removed. The `kam` component is no longer included in new installations, and all references have been removed from the {gitops-title} documentation. link:https://issues.redhat.com/browse/GITOPS-4466[GITOPS-4466]
++
+To remove `kam` resources after an upgrade, run the following command:
```

Comment:
Update is the preferred term per the [term glossary](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update):
```suggestion
To remove `kam` resources after an update, run the following command:
```

Reply:
Fixed.